builtin/providers/dockercloud/resource_dockercloud_service.go

```diff
@@ -0,0 +1,482 @@
+package dockercloud
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/docker/go-dockercloud/dockercloud"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceDockercloudService() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDockercloudServiceCreate,
+		Read:   resourceDockercloudServiceRead,
+		Update: resourceDockercloudServiceUpdate,
+		Delete: resourceDockercloudServiceDelete,
+		Exists: resourceDockercloudServiceExists,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
```

Comment:
@nextrevision One more thing I noticed...there are some missing schema values in `dockercloud_service` that are available in the `go-dockercloud` api https://github.com/docker/go-dockercloud/blob/master/dockercloud/type.go#L413-L432
The only one `go-dockercloud` seems to be missing is `nickname`, which to my knowledge isn't available in the UI yet anyway?